code/modules/projectiles/guns/ballistic.dm

```diff
@@ -120,11 +120,11 @@
 	if (chamber_next_round && (magazine.max_ammo > 1))
 		chamber_round()
 
-/obj/item/gun/ballistic/proc/chamber_round()
+/obj/item/gun/ballistic/proc/chamber_round(keep_bullet = FALSE)
```

Comment:
```suggestion
/obj/item/gun/ballistic/proc/chamber_round(keep_bullet, spin_chamber = TRUE)
```

Reply:
I don't get this one, there isn't any spin functionality in the base class. It would just be a parameter that did nothing.

Reply:
@ShizCalev